packages/dialtone-css/lib/build/less/components/input.less

```diff
@@ -289,6 +289,10 @@
 
     &--right:not(:empty) {
         margin-right: var(--dt-space-400);
+
+        button {
+            margin-right: var(--dt-space-350-negative);
+        }
```

Comment:
This is only meant to affect  dialtone buttons right? maybe we should do `.d.btn` to be a little safer?